src/inverted-index.js

```diff
@@ -1,49 +1,86 @@
 // Index object
 function Index() {
-  var book = [
-    {
-      "title": "Alice in Wonderland",
-      "text": "Alice falls into a rabbit hole and enters a world full of imagination."
-    },
-
-    {
-      "title": "The Lord of the Rings: The Fellowship of the Ring.",
-      "text": "An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring."
-    }
+  this.fileContents = undefined;
+  this.invertedIndex = {};
+  // stop words to ignore on index creation
+  this.stopWords = ["a", "an", "and", "as", "at", "but", "by", "each", "every", "for",
+    "from", "her", "his", "in", "into", "its", "like", "my", "no", "nor",
+    "of", "off", "on", "onto", "or", "our", "out", "outside", "over", "past",
+    "since", "so", "some", "than", "that", "the", "their", "this", "to", "up", "with"
   ];
 
-  this.fileContents = [];
-  this.index_ = {};
-  // this.createIndex = function(filepath) {
-  //   fetch(filepath)
-  //     .then(function(response) {
-  //       response.json();
-  //       return response.json();
-  //     });
-  // };
+  this.readFile = function (filepath) {
```

Comment:
**Code Quality - Critical**

This function need not be declared bound to the index object. You can  declare it independently outside this scope so that it's reused when needed and not created every time an index object is created.